src/content/guides/production.md

```diff
@@ -206,7 +207,7 @@ If you decide to try another minification plugin, just make sure your new choice
 
 ## Source Mapping
 
-We encourage you to have source maps enabled in production, as they are useful for debugging as well as running benchmark tests. That said, you should choose one with a fairly quick build speed that's recommended for production use (see [`devtool`](/configuration/devtool)). For this guide, we'll use the `source-map` option in _production_ as opposed to the `inline-source-map` we used in _development_:
+We encourage you to have source maps enabled in production, as they are useful for debugging as well as running benchmark tests. That said, you should choose one with a fairly quick build speed that's recommended for production use (see [`devtool`](/configuration/devtool)). For this guide, we'll use the `source-map` option in _production_ as opposed to the `inline-source-map` we used in the _development_:
```

Comment:
so preceding `in production` looks good to you, but in development should receive a `the` ? I don't get it

Reply:
@EugeneHlushko I must have missed it while reading. Adding in next commit.